### PR TITLE
Create 20161108yishibashi

### PR DESCRIPTION
**Pull request by yishibashi**
(￣ー￣)ｂｸﾞｯ!

### CODE REVIEW

File: 20161108yishibashi [hunk 1]
```diff
@@ -0,0 +1,18 @@
+title = "ご飯だよ！ 〜 Go Hands-on 〜　に参加しました"
+description = ""
+author = "yishibashi"
+date = "2016-11-08"
+categories = ["go", "prp"]
```
Comment (mikoim) [inline 20161108yishibashi]: prpは要らないっす

File: 20161108yishibashi [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+tags = [
+     "go",
+     "prp",
```
Comment (mikoim) [inline 20161108yishibashi]: 同上

**Comment by yishibashi** [comment]
fixed

**Comment by yishibashi** [comment]
year-month-date_name

ex) 20161108_yishibashi

**Comment by mikoim** [comment]
△ 20161108_yishibashi
○ 20161108_yishibashi**.md**

**Comment by mikoim** [comment]
@yishibashi Before merging this pull request, these commits must be squashed.

**Comment by yishibashi** [comment]
How to squash???

**Comment by mikoim** [comment]
@yishibashi demo uploaded. check Slack channel.

File: src/20161108_yishibashi.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+author: "yishibashi"
```
Comment (mikoim) [inline src/20161108_yishibashi.md]: Use latest front matter style
https://raw.githubusercontent.com/PRParty/weekly/master/src/template.md

File: src/20161108_yishibashi.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+title: "ご飯だよ！ 〜 Go Hands-on 〜　に参加しました"
+slug "go-hands-on"
+description: ""
+date: "2016-11-08"
+tags: ["go"]
+---
+
+- したこと
```
Comment (mikoim) [inline src/20161108_yishibashi.md]: It is better to rewrite like following lines.

```markdown
## したこと

- A Tour of GOの Methods and interfaces の 10/26 まで。
- ところどころ（ポインタ、インターフェースなど）はいまいち理解できなかった。
```

**Comment by mikoim** [comment]
nice :+1:

**Pull request merged**